server/src/main/java/org/opensearch/cluster/routing/allocation/DiskThresholdMonitor.java

```diff
@@ -380,9 +380,17 @@ public void onNewInfo(ClusterInfo info) {
         if ((state.getBlocks().hasGlobalBlockWithId(Metadata.CLUSTER_CREATE_INDEX_BLOCK.id()) == false)
             && nodes.size() > 0
             && nodesOverHighThreshold.size() == nodes.size()) {
+            logger.warn(
+                "Enabling index create block on cluster as all nodes are breaching high disk watermark."
+                    + "Number of nodes above high watermark: {}. Total numbers of node: {}",
```

Comment:
Need a space at end of previous string or start of this one.

Reply:
Addressed the comments.